WebAPI/WebAPI/Models/Employee.cs

```diff
@@ -29,5 +36,12 @@ public class Employee
         // Navigation Properties
         public  virtual Department Department { get; set; }
 
+        [ForeignKey("Shift")]
+        public string ShiftId { get; set; }
+        public virtual Shift Shift { get; set; }
+
+        public DateOnly? LeftDate { get; set; }
+
+
```

Comment:
Please add the fields that we agreed on 1000 times before:

```csharp
public DateTime CreatedAt { get; set; }
public DateTime? LastModified { get; set; }
public SimpleUserDto createdBy { get; set; }
```


![image](https://github.com/habibaelsayed/hr-management-system-enhancement/assets/165601426/daa405a5-222e-4651-9a91-a6da8c1be341)